packages/addons/addon-depends/system-tools-depends/vim/package.mk

```diff
@@ -35,3 +37,9 @@ pre_configure_target() {
 make_target() {
   :
 }
+
+post_makeinstall_target() {
+  cd $INSTALL/storage/.kodi/addons/virtual.system-tools/data/vim/vim*
+  rm -r doc tutor gvimrc_example.vim
+  mv vimrc_example.vim ../vimrc
```

Comment:
I would enclose this in a subshell so that you don't leave the process in a random directory, ie.
```
post_makeinstall_target() {
  (
    cd $INSTALL/storage/.kodi/addons/virtual.system-tools/data/vim/vim*
    rm -r doc tutor gvimrc_example.vim
    mv vimrc_example.vim ../vimrc
  )
}

Reply:
Doesn't it run in a subshell already?

Reply:
No, these functions are executed within `scripts/build` and although it's unlikely to cause an issue in `scripts/build` there may be other later functions that have assumed the process is located in a particular directory so it's better to be safe than sorry and use absolute paths (avoiding the need to `cd`), pushd/popd pwd (eugh) or - my preference - use a sub-shell so that whatever you do in the function has absolutely no effect on any code that executes once you leave the function.